zed/src/main.cpp

```diff
@@ -199,6 +204,7 @@ class zed_imu_thread : public threadloop {
     ullong imu_time;
 
     std::size_t last_serial_no {0};
+    long long _imu_integrator_seq{0};
```

Comment:
Are we okay with using `long long` instead of `int64_t` or something along those lines?